test/command_callback/test_llc_command_callback.vader

```diff
@@ -0,0 +1,29 @@
+Before:
+  Save g:ale_llvm_llc_executable
+
+  unlet! g:ale_llvm_llc_executable
+  unlet! b:ale_llvm_llc_executable
+
+  runtime ale_linters/llvm/llc.vim
+
+After:
+  unlet! g:ale_llvm_llc_executable
+  unlet! b:ale_llvm_llc_executable
+  Restore
+
+Execute(llc command is customizable):
+  let cmd = ale_linters#llvm#llc#GetCommand(bufnr(''))
+  AssertEqual stridx(cmd, ale#Escape('llc')), 0
+
+  let g:ale_llvm_llc_executable = 'llc-5.0'
+  let cmd = ale_linters#llvm#llc#GetCommand(bufnr(''))
+  AssertEqual stridx(cmd, ale#Escape('llc-5.0')), 0
+
+  let b:ale_llvm_llc_executable = 'llc-4.0'
+  let cmd = ale_linters#llvm#llc#GetCommand(bufnr(''))
+  AssertEqual stridx(cmd, ale#Escape('llc-4.0')), 0
+
+Execute(space can be contained in llc executable path):
+  let b:ale_llvm_llc_executable = '/path/space contained/llc'
+  let cmd = ale_linters#llvm#llc#GetCommand(bufnr(''))
+  AssertEqual stridx(cmd, ale#Escape('/path/space contained/llc')), 0
```

Comment:
I will probably end up changing these tests later, as the failure message will print `-1 != 0`, which doesn't tell you much of anything.

Reply:
You're correct. But how can I write a test to check the string is prefixed? Is there any way to specify assertion message? I'm not familiar with vader.vim. So could you please help to improve these tests?

Reply:
Ah, I found that message can be specified. I'll improve message. https://github.com/junegunn/vader.vim#execute